### PR TITLE
:bricks: walking skeleton

### LINKED ISSUE #19: 專案基礎建設
# Clean Architecture 
- Domain Layer
  - Entities with Domain Logic
- Application Layer
  - Use Cases
  - Domain Event Bus
- Interface Adapter Layer
  - Web API (Sprint Boot)
    - RESTful API
    - Web Socket
  - Repository (MongoDb)

### PR DESCRIPTION
**Pull request by frankvicky**
## Why need this change? / Root cause: 
- So everyone can start develop.
## Changes made:
- Whole project
## Test Scope / Change impact:
- Whole project
## Issue
- resolved #19

### CODE REVIEW

**Review by Wally5077 — APPROVED** [review]
LGTM

File: domain/pom.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>Lobby-Platform-Domain</artifactId>
+
+    <parent>
+        <groupId>waterballsa.gaas</groupId>
+        <artifactId>Lobby-Platform-Root</artifactId>
```
Comment (kuoche1712003) [inline domain/pom.xml:9]: 這邊的 artifactId 跟 root 一樣是刻意的嗎？

**Comment by kuoche1712003** [comment]
artifactId 大寫看起來很不習慣 XD

**Review by ricksu978 — APPROVED** [review]
基礎建設，沒大問題就先 merge。
小步快跑

**Pull request merged**